addOns/network/src/main/javahelp/help/contents/options/clientcertificates.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>Client Certificates</TITLE>
+</HEAD>
+<BODY>
+	<H1>Client Certificates</H1>
```

Comment:
I did not move the core help content here. Please, take a look and let me know if what's there needs to be moved.

Reply:
It's being handled the same way as the other options, with the `-config` args.

Reply:
Okay, I think I see parts were renamed store vs persist which might be why it didn't stick out as much and since they aren't specific CLI options anymore I guess it doesn't need specific help entries.